SWLOR.Game.Server/Feature/ShipModuleDefinition/MissileLauncherModuleDefinition.cs

```diff
@@ -88,10 +79,36 @@ private void PerformAttack(uint activator, uint target, int dmg, int attackBonus
                 .ValidTargetType(ObjectType.Creature)
                 .PowerType(ShipModulePowerType.High)
                 .RequirePerk(PerkType.OffensiveModules, requiredLevel)
-                .Recast(recast)
-                .Capacitor(capacitor)
+                .Recast(12f)
+                .ValidationAction((activator, activatorShipStatus, target, targetShipStatus, moduleBonus) =>
+                {
+                    var item = GetItemPossessedBy(activator, "ship_missile");
```

Comment:
Another situation where I'd set "ship_missile" to a constant so you are limiting the use of magic strings.